acceptance/support/common_steps.rb

```diff
@@ -17,28 +17,28 @@ def given_I_am_logged_in
     editor.load
     editor.sign_in_button.click
 
-    if ENV['CI_MODE'].present?
-      sleep(3)
-      expect(page).to have_content('Please select the log in option that matches your work email')
-
-      # Executing javascript directly as the fields and button are hidden on the
-      # login page for the moment
-      editor.execute_script(
-        "document.getElementById('email').value = '#{ENV['ACCEPTANCE_TESTS_USER']}'"
-      )
-      editor.execute_script(
-        "document.getElementById('password').value = '#{ENV['ACCEPTANCE_TESTS_PASSWORD']}'"
-      )
-      editor.execute_script(
-        "document.getElementById('btn-login').click()"
-      )
-    else
-      editor.sign_in_email_field.set('form-builder-developers@digital.justice.gov.uk')
-      editor.sign_in_submit.click
+    using_wait_time 10 do
```

Comment:
if default wait time has gone up to 15 then maybe this is no longer needed?

Reply:
Good point. I've changed this and am running tests locally, which are passing.